docs/api/navigators/StackNavigator.md

```diff
@@ -137,7 +137,7 @@ Color for material ripple (Android >= 5.0 only)
 
 #### `gesturesEnabled`
 
-Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android
+Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android. This value when set is only respected in `mode` == `card`.
```

Comment:
@nihgwu Not sure I am following you here. Are you saying that `gesturesEnabled` should be configured via `StackNavigatorConfig`? Having it in `navigationOptions` allows you to override this setting per screen. To me it's pretty much the same as being able to show/hide the navbar itself.